Libplanet.Explorer.Executable/Options.cs

```diff
@@ -130,6 +130,14 @@ public string IceServerUrl
 in memory version is used.")]
         public string StorePath { get; set; }
 
+        [Option(
+            'T',
+            "store-type",
+            Default = null,
+            HelpText = @"The type of the blockchain store. If omitted (default)
+in DefaultStore is used.")]
```

Comment:
It should show what type can this option take here, e.g., `rocksdb` (not `RocksDBStore`).  It would be way better if we can enumerate these identifiers without code dup between here and the `switch` statement in `Program.LoadStore()` method…

Reply:
I have no idea to implement your suggestion with C#. Because the argument used as one of `Attribute`'s parameters, must be a constant value. It means there is no way to give the types to `HelpText`, whether we manage them as string array or enum.

But, if we moved to [mayuki/Cocona], then it seems able to be applied successfully. please see [the sample project][shell-completion-candidates].

[mayuki/Cocona]: https//github.com/mayuki/Cocona
[shell-completion-candidates]: https://github.com/mayuki/Cocona/blob/master/samples/Advanced.ShellCompletionCandidates/Program.cs

Reply:
> I have no idea to implement your suggestion with C#. Because the argument used as one of `Attribute`'s parameters, must be a constant value. It means there is no way to give the types to `HelpText`, whether we manage them as string array or enum.

Unless `CommandLine` provides imperative API for this, I also believe it's impossible to achieve my suggestion without replacing `CommandLine` with other one.